.buildkite/pipelines/pull-request/part-1-arm.yml

```diff
@@ -0,0 +1,13 @@
+config:
+  allow-labels: "test-arm"
+steps:
+  - label: part-1-arm
+    command: .ci/scripts/run-gradle.sh -Dignore.tests.seed checkPart1
+    timeout_in_minutes: 300
+    agents:
+      provider: aws
+      imagePrefix: elasticsearch-ubuntu-2004-aarch64
```

Comment:
Is this the name of the elasticsearch image? `2004` looks a bit old!, but I suspect that the version of Ubuntu is not actually from 2004! ? ;-)

Reply:
no it's the ubuntu image we use. ubuntu 20.04 lts

Reply:
ha! ;-)